core/src/com/unciv/ui/tilegroups/CityButton.kt

```diff
@@ -155,6 +159,52 @@ class CityButton(val city: CityInfo, internal val tileGroup: WorldTileGroup, ski
         parent.addAction(floatAction)
     }
 
+    private fun getPopulationGroup(showGrowth: Boolean): Group {
+        val group = Group()
+
+        val populationLabel = city.population.population.toString().toLabel()
```

Comment:
populationLabel should be colored using the secondary color of the civ owning the city:
![Off2](https://user-images.githubusercontent.com/17114100/71821329-23ab2a80-3092-11ea-82a4-136e8c3b1781.png)